### PR TITLE
Fixing the values in enums.py:UI

### PR DESCRIPTION
**Pull request by pbajjuri20**
Upstream Kiali has updated the values.

upstream_Kiali

### CODE REVIEW

**Comment by kialiqeci** [comment]
Can one of the admins verify this patch?

**Comment by hhovsepy** [comment]
@pbajjuri20 can you also fix the HIDE and RESPONSE_TIME values as well, according to your screenshot.

**Comment by pbajjuri20** [comment]
@hhovsepy: I will update the HIDE to NO_LABEL, but the RESPONSE_TIME value is not changed.

File: kiali_qe/components/enums.py [hunk 1]
```diff
@@ -145,9 +145,9 @@ class GraphPageBadgesFilter(StringEnum):
 
 
 class EdgeLabelsFilter(StringEnum):
-    HIDE = ('Hide')
-    REQUEST_PER_SECOND = ('Requests per second')
-    REQUEST_PERCENT = ('Requests percent of total')
+    NO_LABEL = ('No label')
+    REQUEST_RATE = ('Requests rate')
+    REQUEST_DISTRIBUTION = ('Requests distribution')
     RESPONSE_TIME = ('Response time 95th percentile')
```
Comment (hhovsepy) [inline kiali_qe/components/enums.py:151]: It does not contain '95th percentile' anymore.

Comment (pbajjuri20) [reply kiali_qe/components/enums.py]: updated the value

**Pull request merged**